docs/machine-learning/resources/transforms.md

```diff
@@ -1,168 +1,102 @@
+| <xref:Microsoft.ML.TransformExtensionsCatalog.Concatenate%2A> | Concatenate one or more input columns into a new output column |
+| <xref:Microsoft.ML.TransformExtensionsCatalog.CopyColumns%2A> | Copy and rename one or more input columns |
+| <xref:Microsoft.ML.TransformExtensionsCatalog.DropColumns%2A> | Drop one or more input columns |
+| <xref:Microsoft.ML.TransformExtensionsCatalog.SelectColumns%2A> | Select one or more columns to keep from the input data |
 
-## Label parsing
+## Normalization and scaling
 
 | Transform | Definition |
 | --- | --- |
-| <xref:Microsoft.ML.Transforms.LabelConvertTransform> |  Converts labels. |
-| <xref:Microsoft.ML.Transforms.LabelIndicatorTransform> | Remaps multiclass labels to binary True, False labels, primarily for use with OVA.|
+| <xref:Microsoft.ML.NormalizationCatalog.NormalizeMeanVariance%2A> | Subtract the mean (of the training data) and divide by the variance (of the training data) |
+| <xref:Microsoft.ML.NormalizationCatalog.NormalizeLogMeanVariance%2A> | Normalize based on the logarithm of the training data |
+| <xref:Microsoft.ML.NormalizationCatalog.NormalizeLpNorm%2A> | Scale input vectors by their [lp-norm](https://en.wikipedia.org/wiki/Lp_space#The_p-norm_in_finite_dimensions), where p is 1, 2 or infinity. Defaults to the l2 (Euclidean distance) norm |
+| <xref:Microsoft.ML.NormalizationCatalog.NormalizeGlobalContrast%2A> | Normalize based on the global contrast of the training data |
+| <xref:Microsoft.ML.NormalizationCatalog.NormalizeBinning%2A> | Assign the input value to a bin index and divide by the number of bins to produce a float value between 0 and 1. The bin boundaries are calculated to evenly distribute the training data across bins |
+| <xref:Microsoft.ML.NormalizationCatalog.NormalizeSupervisedBinning%2A> | Assign the input value to a bin based on its correlation with label column |
+| <xref:Microsoft.ML.NormalizationCatalog.NormalizeMinMax%2A> | Scale the input by the difference between the minimum and maximum values in the training data |
 
-## Missing values
+## Data type transformations
 
 | Transform | Definition |
 | --- | --- |
-| <xref:Microsoft.ML.Transforms.MissingValueDroppingTransformer> | Drops missing values from columns. |
-| <xref:Microsoft.ML.Transforms.MissingValueIndicatorTransform> | Creates a boolean output column with the same number of slots as the input column, where the output value is true if the value in the input column is missing. |
-| <xref:Microsoft.ML.Transforms.MissingValueReplacingTransformer> | Handle missing values by replacing them with either the default value or the mean/min/max value (for non-text columns only). |
+| <xref:Microsoft.ML.ConversionsExtensionsCatalog.ConvertType%2A> | Convert the type of an input column to a new type |
+| <xref:Microsoft.ML.ConversionsExtensionsCatalog.MapValue*> | Create a dictionary of key to value mappings |
```

Comment:
Maps the values keys(categories), either by creating the mapping from the data of the column to transform, or by using a given dictionary of